spec/21-http_response_header_format.md

```diff
@@ -144,16 +131,16 @@ traceparent: 00-4bf92f3577b34da6a3ce929d0e0e4736-d75597dee50b0cac-01
 ```
 Response
 ```http
-traceresponse: 00-1baad25c36c11c1e7fbd6d122bd85db6--01
+traceresponse: 00-1baad25c36c11c1e7fbd6d122bd85db6-cab70b47728a8a99-01
 ```
 
-In this example, a participant in a trace with ID `4bf92f3577b34da6a3ce929d0e0e4736` calls a third party system that collects their own internal telemetry using a new trace ID `1baad25c36c11c1e7fbd6d122bd85db6`. When the third party completes its request, it returns the new trace ID and internal sampling decision to the caller. If there is an error with the request, the caller can include the third party's internal trace ID in a support request.
+In this example, a participant in a trace with ID `4bf92f3577b34da6a3ce929d0e0e4736` calls a third party system that collects their own internal telemetry using a new trace ID `1baad25c36c11c1e7fbd6d122bd85db6`. When the third party completes its request, it returns the new trace ID, the ID of the operation, and internal sampling decision to the caller. If there is an error with the request, the caller can include the third party's internal trace ID in a support request.
 
-**Note**: In this case, the `proposed-parent-id` was omitted from the response because, being a part of a different trace, it was not necessary for the caller.
+### Tail Sampling Load Balancer
```

Comment:
Tail sampling has a pretty well defined meaning in the industry already, and it's not what is described here. I would rename the section, maybe "backtrace"

Reply:
how did you come up with the term backtrace? It isn't immediately obvious to me what the connection is. Backtrace to me would imply stitching the trace using response headers, which is not what we're doing.

Reply:
What about "deferred"? We are already using it in other parts of the document.

https://www.w3.org/TR/trace-context/#x3-2-2-5-1-sampled-flag:~:text=If%20a%20component%20deferred%20or%20delayed,trace%20is%20initiated%20by%20this%20component

https://www.w3.org/TR/trace-context/#x3-2-2-5-1-sampled-flag:~:text=Deferred%20sampling

Reply:
yes, deferred sampling sounds good here